torchvision/models/detection/faster_rcnn.py

```diff
@@ -383,15 +383,15 @@ def fasterrcnn_resnet50_fpn(
             Valid values are between 0 and 5, with 5 meaning all backbone layers are trainable. If ``None`` is
             passed (the default) this value is set to 3.
     """
-    trainable_backbone_layers = _validate_trainable_layers(
-        pretrained or pretrained_backbone, trainable_backbone_layers, 5, 3
-    )
+    is_trained = pretrained or pretrained_backbone
```

Comment:
nit: if the model was trained for detection with large batch sizes from scratch, and then we finetune it afterwards (still with large batch sizes) then in this case we would be using `FrozenBatchNorm`.

This is an ok heuristic, but hints that we might want to make this an explicit parameter from the constructor in the future